### PR TITLE
Feat/core changes angular update v18

### PR DESCRIPTION
**Pull request by x3mlestat**

### CODE REVIEW

**Review by denisogolubev — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "xm-webapp",
-  "version": "6.0.208",
+  "version": "6.0.209",
```
Comment (sergeysenja1992) [inline package.json:3]: pls update major version to 7

Comment (x3mlestat) [reply package.json]: fixed

**Review by LevBD — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -53,19 +53,19 @@
   "dependencies": {
     "@ajsf/core": "0.8.0",
     "@ajsf/material": "0.8.0",
-    "@angular/animations": "17.0.4",
-    "@angular/cdk": "17.0.4",
-    "@angular/common": "17.0.4",
-    "@angular/compiler": "17.0.4",
-    "@angular/core": "17.0.4",
+    "@angular/animations": "18.2.7",
+    "@angular/cdk": "^18.2.7",
```
Comment (sergeysenja1992) [inline package.json:57]: pls, don`t use version with `^`

File: tsconfig.json [hunk 3]
```diff
@@ -18,7 +18,8 @@
     ],
     "lib": [
       "es2020",
-      "dom"
+      "dom",
+      "ES2021.String"
```
Comment (sergeysenja1992) [inline tsconfig.json:22]: how about ES15 ?

File: package.json (hunk 2 above)
Comment (x3mlestat) [reply package.json]: fixed

**Review by sergeysenja1992 — APPROVED** [review]

**Pull request merged**